IdentityServer/v7/docs/content/quickstarts/3_api_access.md

```diff
@@ -102,29 +98,33 @@ builder.Services.AddAuthentication(options =>
     });
 ```
 
-Since *SaveTokens* is enabled, ASP.NET Core will automatically store the id,
+Since _SaveTokens_ is enabled, ASP.NET Core will automatically store the id,
```

Comment:
This still mentions refresh token. When we add a new QS, this one should be completely ignorant of refresh tokens and token management.

Reply:
Fixed